app/assets/javascripts/application.js

```diff
@@ -125,6 +125,8 @@ function onContentLoad(){
   $("#new_lookup_keys_ select").select2('destroy');
   // Interface selects are initiated separately
   $("#interfaces select").select2('destroy');
+  // Compute profiles additional nics & volumes are initiated separately
+  $(".nics_fields_template select, .volumes_fields_template select").select2('destroy');
```

Comment:
``` $(".nics/volumes_fields_template select")``` are using the same [code base in lookup_keys](https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/lookup_keys.js#L83), so they don't need to be initialized here